build/configs/artik053/README.md

```diff
@@ -198,7 +198,42 @@ you must modify partition_map.cfg matching the sizes in ARTIK053_FLASH_PART_LIST
 
 will be updated
 
+#### audio
```

Comment:
#### audio
This can be used to test audio functionality.
xxx xxxx xxx are enabled.

#### extra
This can be used to use full functionality of Tizen RT on ARTIK053.
xxx, xxx, xxx are enabled

#### hello
This provides simple hello world application including kernel.

#### iotivity
This can be used to use Iotivity (OCF).

blah blah